packages/markup-frontstage/src/services/SavedView.ts

```diff
@@ -147,22 +219,125 @@ export const createSavedViewData = (vp: Viewport): SavedViewData => {
 
   const data: SavedViewData = {
     alwaysDrawn,
-    neverDrawn,
     categories,
+    changeSetId,
+    displayStyleProps: JSON.stringify(viewState.displayStyle.toJSON()),
+    iModelId,
+    is2d: false,
     models,
-    version,
+    neverDrawn,
+    perModelCategoryVisibility,
     projectId,
-    iModelId,
-    changeSetId,
     sourceId,
-    userView: true,
     state,
-    displayStyleProps: JSON.stringify(viewState.displayStyle.toJSON()),
-    perModelCategoryVisibility,
+    userView: true,
+    version,
   };
 
   // Parse all data from emphasize elements (e.g. colorization, isolates, hidden, emphasize)
   parseFromEmphasizeElements(vp, data);
 
   return data;
 };
+
+/**
+ * Determines whether or not given saved view is spatial.
+ * @param view name of the Saved View.
+ */
+export const isSpatialSavedView = (view: SavedViewData) => {
+  return (view.is2d === undefined || !view.is2d) && "models" in view;
```

Comment:
shouldn't !view.is2d suffice here? That should cover undefined and false

Reply:
added !!view.is2d to cover undefined as well.